autoxtrabackup.py

```diff
@@ -122,54 +122,55 @@ def validate_file(file):
               expose_value=False,
               is_eager=True,
               help="Version information.")
-@click.option('--defaults_file',
+@click.option('--defaults-file',
```

Comment:
noooooooooooo :'(

why not a simpler "--config"??

also it should have been "default" and not "defaults"

Reply:
Because this tool is for MySQL world and with MySQL you are going to specify config file with `--defaults-file` option :)
```
$ mysql --help | grep defaults
--print-defaults        Print the program argument list and exit.
--no-defaults           Don't read default options from any option file,
--defaults-file=#       Only read default options from the given file #.
--defaults-extra-file=# Read this file after the global files are read.
--defaults-group-suffix=#
```

Reply:
o_0 this is the worst name i ever seen and it's on mysql

i'm shocked to see such a shit name

Reply:
ahahah :) yes for your surprise it is for specifying config file with MySQL also.
So why we are using config files? To specify options and default values right?
--defaults-file -> the file where you can put default(s) values for your options.

Reply:
i'm so sad :(

Reply:
:) The MySQL is cruel as life

Reply:
true